frontend/src/data/Breakdowns.ts

```diff
@@ -44,23 +44,29 @@ function demographicBreakdownToCol(
 
 export class Breakdowns {
   geography: GeographicBreakdown;
-  // Note: this assumes only one demographic breakdown at a time. If we want to
-  // support more later we can refactor this to multiple boolean fields.
-  demographic?: DemographicBreakdown;
```

Comment:
Is the DemographicBreakdown type still used anywhere or can we delete it?

Reply:
It was used in one other place but I don't think it was really needed- just removed!